build/charts/antrea/crds/flowexporterdestination.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: flowexporterdestinations.crd.antrea.io
+  labels:
+    app: antrea
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+      - name: Address:Port
+        type: string
+        description: Address of flow collector.
+        jsonPath: .spec.address
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - address
+                - protocol
+              properties:
+                address:
+                  type: string
+                  description: >
+                    The flow collector address including port as a string.
+
+                    Example:
+                    - flow-aggregator/flow-aggregator:14739
+                    - 10.244.10.10:4739
+                  pattern: ^.+:[0-9]+$
+                protocol:
+                  type: object
+                  description: >
+                    The protocol used to send flow details.
+
+                    Exactly one must be defined and non-nil.
+                  oneOf:
+                  - required: [ipfix]
+                  - required: [grpc]
+                  properties:
+                    ipfix:
+                      type: object
+                      description: Configuration for using IPFIX protocol.
+                      required:
+                      - transport
+                      properties:
+                        transport:
+                          type: string
+                          enum:
+                          - tcp
+                          - udp
+                          - tls
+                    grpc:
+                      type: object
+                      description: Configuration for using gRPC protocol.
+                filter:
+                  type: object
+                  properties:
+                    protocols:
+                      type: array
+                      description: >
+                        Filter for only flows whose protocol which match this filter.
+                        The default is accept all protocols if unset or nil.
+
+                        Supported values are [tcp, udp, icmp, sctp].
+                      items:
+                        type: string
+                        enum:
+                        - tcp
+                        - udp
+                        - icmp
```

Comment:
When was ICMP support added? I don't see it in `pkg/agent/flowexporter/filter/protocol_filter.go`